sdk/iOS/src/MSTableOperation.h

```diff
@@ -28,13 +28,13 @@ typedef NS_OPTIONS(NSUInteger, MSTableOperationTypes) {
 @property (nonatomic, readonly) MSTableOperationTypes type;
 
 /// The name of the table associated with the item
-@property (nonatomic, copy, readonly) NSString *tableName;
+@property (nonatomic, copy, readonly, nonnull) NSString *tableName;
 
 /// The Id of the item the operation should run on.
-@property (nonatomic, copy, readonly) NSString *itemId;
+@property (nonatomic, copy, readonly, nonnull) NSString *itemId;
 
 /// The item that will be sent to the server when execute is called.
-@property (nonatomic, strong) NSDictionary *item;
+@property (nonatomic, strong, nonnull) NSDictionary *item;
```

Comment:
This one should be nullable, the internal constructor can make one w/o an item, so it is not 100% always around.